### PR TITLE
Fix popper.js library missing

### PR DESCRIPTION
**Pull request by anpolimus**
2nd level of navigation was not working at Carnation. System was not able to find Poppeer.js library.

## Steps for review

- [x] Open carnation on mobile.
- [x] Click on hamburger menu.
- [ ] Check that you can open Programs menu

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Review by podarok — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3215/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3215/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3215/

**Comment by anpolimus** [comment]
@podarok is it possible to test it on a build?
I see that our build link returns 404

**Comment by podarok** [comment]
retest this please

**Comment by podarok** [comment]
> @podarok is it possible to test it on a build?
> I see that our build link returns 404

yes, sure. Builds are wiped out overnight or due to lack of space, so it's better to test at the same day or next day with asking bot to rebuild

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3216/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3216/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3216/

**Comment by podarok** [comment]
image

**Comment by anpolimus** [comment]
@podarok yes we saw it at our project as well and were able to fix it in a subtheme.
Now I have to find a solution how to back port it.

**Comment by podarok** [comment]
popper is here 
image

File: themes/openy_themes/openy_carnation/openy_carnation.libraries.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ global-styling:
   js:
     dist/main.js: {}
     https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.1/js/bootstrap.min.js: {type: external, minified: true}
+    dist/js/popper.min.js: {}
```
Comment (NightHunterSV) [inline themes/openy_themes/openy_carnation/openy_carnation.libraries.yml]:
```suggestion
    //cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js: {type: external, minified: true}
```

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3238/

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3239/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3239/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3239/

**Comment by podarok** [comment]
Seems like it does work 
image
Have you faced with any issues due to upgraded version of dependency in the theme? @anpolimus @NightHunterSV

**Comment by anpolimus** [comment]
@podarok I've faced with it during the new build.
We are facing it at VY builds last time.
It looks like it came couple weeks ago.

**Pull request merged**